BLE_SupportedFeatures/readme.md

```diff
@@ -0,0 +1,22 @@
+# BLE Supported Features
+
+This application tells you which optional BLE features are supported by your device. This info can be useful as product datasheets often do not go into these details.
+Knowing which optional features are supported might impact how exactly you implement BLE feature of your applications.
+This demo also showcases how to access BLE related API methods via your BLE object instance.
```

Comment:
I think it would be enough for this example to init and show features, the advertising is already showcases in the dedicated example.

Reply:
That is right, the advertising part of this example is not necessary. Just thought that no BLE examples make sense without any type of advertising at least.